### PR TITLE
Feat/#1 hero component

### PR DESCRIPTION
**Pull request by Jaimeandres97**
Please review the layout of the hero components for mobile and desktop

### CODE REVIEW

**Comment by juancho11gm** [comment]
Please squash your commits (So we only have 1 commit). I think your commit must be named with the `feat` category and must attach the issue number.

**Review by juancho11gm — COMMENTED** [review]
What are Zone.Identifier files used for?

File: css/desktop.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    max-width: 18rem;
+    margin-bottom: 2.5rem;
+}
+
+.description {
+    font-size: 4rem;
+    margin-bottom: 2.4rem;
+}
+
+.primary-button .quiero-combo{
```
Comment (juancho11gm) [inline css/desktop.css]: please use English only

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+:root {
+    font-size: 62.5%;
+    --text-input-field: #f9f9f9;
+    --black: #000000;
+    --orange: rgb(238, 71, 5);
+    --sm: 1em;
+    --md: 1.6em;
+    --lg: 2.6em;
```
Comment (juancho11gm) [inline css/style.css]: what do these values mean? I know they are for different viewports but I don't know if they are used for box model properties, positioning, font size...

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+html {
+    font-family: 'Sen', sans-serif;
+}
+
+:root {
+    font-size: 62.5%;
+    --text-input-field: #f9f9f9;
+    --black: #000000;
+    --orange: rgb(238, 71, 5);
```
Comment (juancho11gm) [inline css/style.css]: You are mixing Hexa values and rgb values. Please use just one of them, just to be consistent. I would prefer Hexa values.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <section class="hero-header"> 
+        <div class="hero-text">
+            <img src="./img/star-logo.svg" alt="Star + Logo" class="star-logo"/>
+            <h2 class="description">Más de lo que te gusta.</h2>
+            <div class="primary-button">
+                <button class="primary-button quiero-combo">QUIERO COMBO+</button>
+            </div>
+            <a href="/" class="anchor-subscription">Suscribirme solo a Star+</a>
+        </div>
+    </section>
```
Comment (juancho11gm) [inline index.html]: Nice, let's start using [BEM](http://getbem.com/introduction/). We will have benefits in the near future 😉

**Comment by juancho11gm** [comment]
I think you can include the `min-height: 100vh;` for the Hero. Just as the star + website does

File: css/desktop.css (hunk 1 above)
Comment (juancho11gm) [inline css/desktop.css]: With [linear gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient()), you can get the button's background-color

`background: linear-gradient(90deg,#fa1660,#ff6e00);`

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    border: 0;
+}
+
+html {
+    font-family: 'Sen', sans-serif;
+}
+
+:root {
+    font-size: 62.5%;
+    --text-input-field: #f9f9f9;
```
Comment (juancho11gm) [inline css/style.css]:
```suggestion
    --text-input-field: #F9F9F9;
```

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    margin: 0;
+    border: 0;
+}
+
+html {
+    font-family: 'Sen', sans-serif;
+}
+
+:root {
+    font-size: 62.5%;
```
Comment (juancho11gm) [inline css/style.css]: Let's use the star plus font sizes instead of percentages

File: css/desktop.css (hunk 1 above)
Comment (juancho11gm) [reply css/desktop.css]: please update the quiero-combo class

File: index.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    <section class="hero-header"> 
+        <div class="hero-text">
+            <img src="./img/star-logo.svg" alt="Star + Logo" class="star-logo"/>
+            <h2 class="description">Más de lo que te gusta.</h2>
+            <div class="primary-button">
+                <button class="primary-button combo-button">QUIERO COMBO+</button>
+            </div>
+            <a href="/" class="anchor-subscription">Suscribirme solo a Star+</a>
+        </div>
+    </section>
```
Comment (juancho11gm) [inline index.html]:
```suggestion
    <section class="hero"> 
        <div class="hero__content">
            <img class="hero__image" src="./img/star-logo.svg" alt="Star + Logo" "/>
            <h2 class="hero__title">Más de lo que te gusta.</h2>
            <div class="hero__button-container">
                <button class="hero__button">QUIERO COMBO+</button>
            </div>
            <a class="hero__link" href="/" >Suscribirme solo a Star+</a>
        </div>
    </section>
```

Please check this article https://scalablecss.com/bem-nesting-grandchild-elements/

File: css/tablet.css [hunk 8]
```diff
@@ -0,0 +1,30 @@
+.hero-header {
+    display: grid;
+    place-items: center;
+    background-image: url('/img/hero-md.jpg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
```
Comment (juancho11gm) [inline css/tablet.css]: Please use the same box model values as the star channel website does
Screen Shot 2022-02-27 at 6 38 50 PM

Comment (juancho11gm) [reply css/tablet.css]: Look what happen in the tablet viewport.

Screen Shot 2022-02-27 at 6 39 49 PM

**Comment by Jaimeandres97** [comment]
Please review the changes, I didn't change the button color because I liked the orange gradient. Maybe there are some small details that I would like to ask you to make it perfect. Thanks

**Comment by juancho11gm** [comment]
I like it. Nice job, in the next PR make sure to apply a background-color to the body. I'm creating the next ticket 👍

**Comment by juancho11gm** [comment]
And please mark as resolve all the comments

**Review by juancho11gm — APPROVED** [review]

**Pull request merged**